go/interpreter/lfvm/interpreter_test.go

```diff
@@ -555,6 +555,25 @@ func TestSteps_FailsWithLessGasThanStaticCost(t *testing.T) {
 	}
 }
 
+func TestInterpreter_InstructionsFailWhenExecutedInRevisionsEarlierThanIntroducedIn(t *testing.T) {
+	for _, op := range allOpCodes() {
+		minRev := _introducedIn.get(op)
```

Comment:
nit: `minRevision` is used as the max boundary for the `for` loop in the line below. Is it min revision or max for the loop? maybe `introducedRevision` or some other name would be a little nicer.